tests/cli.rs

```diff
@@ -10,6 +10,34 @@ use tempfile::NamedTempFile;
 // - Figure out how to mock tmux or use optional env var in main when testing
 // for its presence.
 
+#[test]
+fn it_returns_the_expected_debug_output() -> Result<(), Box<dyn std::error::Error>> {
+    let mut file = NamedTempFile::new()?;
+    writeln!(
+        file,
+        "name = \"debug\"
+        [[windows]]
+          name = \"one\"
+        [[windows]]
+          name = \"two\"
+        "
```

Comment:
Doing a raw string literal here would save you from having to escape all the double quotes.